packages/help-center/src/components/help-center-search.scss

```diff
@@ -148,6 +148,14 @@
 						display: block;
 						align-self: center;
 					}
+					.help-center-search-results__content {
+						width: 100%;
+						.help-center-search-results__badge {
+							font-size: $font-body-extra-small;
+							text-transform: uppercase;
+							text-align: center;
```

Comment:
Nit: the text doesn't wrap regardless of string length so you don't really need `text-align: center`. But it might make sense to still make the badge smaller like you did before because it is a very tight space!

Very much up to you, but I'd try something like this:

```suggestion
							height: auto;
							font-size: $font-body-extra-small;
							padding: 0 8px;
							text-transform: uppercase;
```

In my view, just makes it fit a bit more nicely:

<img width="394" alt="Screenshot 2024-02-26 at 09 41 33" src="https://github.com/Automattic/wp-calypso/assets/43215253/15cf7571-8e26-46e5-9aa4-4dd8f88dd52e">

Reply:
Thank you, looks smother.